content/chainguard/chainguard-images/working-with-images/compiled-programs.md

```diff
@@ -0,0 +1,79 @@
+
+The following packages are included in the `static:latest` Chainguard Image:
+`ca-certificates-bundle`, `chainguard-baselayout`, `glibc-locale-posix`, `tzdata`, and `wolfi-baselayout`.
+
+### glibc-dynamic
+The `glibc-dynamic` Chainguard Image is best suited where you need to host dynamically linked binaries which depend on the C standard library. Like the `static` image, `glibc-dynamic` is intended to be used as a base image only, and you must add your own binaries to the image.
+
+The `glibc-dynamic` image is freely available in two variants: `:latest` and `:latest-dev`. The `:latest-dev` image adds additional packages which are not present in `:latest` to help facilitate application development. It is suggested to use the `:latest` image for production-facing purposes because of its smaller footprint. 
+
+The following packages are included in the `glibc-dynamic:latest` Chainguard Image:
+`ca-certificates-bundle`, `chainguard-baselayout`, `glibc`, `glibc-locale-posix`, `ld-linux`, `libgcc`, `libstdc++`, and `wolfi-baselayout`.
+
+### cc-dynamic
+The `cc-dynamic` Chainguard Image is deprecated. It is suggested that you use the `glibc-dynamic` image instead, as it is designed to replace `cc-dynamic`.
+
+### gcc-glibc
+The `gcc-glibc` Chainguard Image is best suited for building C applications which depend on `glibc`. There are two freely available variants of this image, `:latest` and `:latest-dev`. `:latest-dev` is a developer variant of the image which adds additional packages such as `bash` to facilitate the development process. 
+
+In comparison to the `static` and `glibc-dynamic` Chainguard Images, `gcc-glibc` is intended to be used to develop programs based on the C standard library, instead of simply hosting binaries. Because of this, it contains additional packages such as `make`, `busybox`, as well as `gcc` to compile programs.
+
+The following packages are included in the `gcc-glibc:latest` Chainguard Image:
+`binutils`, `build-base`, `busybox`, `ca-certificates-bundle`, `gcc`, `glibc`, `glibc-dev`, `glibc-locale-posix`, `gmp`, `isl`, `ld-linux`, `libatomic`, `libcrypt1`, `libgcc`, `libgo`, `libgomp`, `libstdc++`, `libstdc++-dev`, `libxcrypt`, `libxcrypt-dev`, `linux-headers`, `make`, `mpc`, `mpfr`, `nss-db`, `nss-hesiod`, `pkgconf`, `posix-cc-wrappers`, `wolfi-baselayout`, and `zlib`.
+
+### glibc-openssl
+*Paid Image* \
+The `glibc-openssl` Chainguard Image is designed for C applications which depend on OpenSSL. It contains a few packages in addition to those found in the `glibc-dynamic` image to achieve this. It comes in two variants,`latest` and `latest-dev`. As in the aforementioned images, `latest` is designed for deployment, while `latest-dev` contains additional packages to assist in program development such as a shell and package manager.
```

Comment:
" It contains a few packages in addition to those found in the glibc-dynamic image to achieve this." reads a little clunky, could remove or reword.